src/Agent/MsiInstaller/InstallerActions/CustomActions.cs

```diff
@@ -234,6 +234,11 @@ public ActionResult ValidateFilesAndFolders()
 
             DeleteFile(@"C:\ProgramData\New Relic\.NET Agent\Extensions\netframework\NewRelic.Providers.Wrapper.CastleMonoRail2.Instrumentation.xml");
 
+            // Asp35 was renamed to AspNet
```

Comment:
Removing this file from the `netcore` (.NET Core/5+ agent) folder should be unnecessary since this wrapper is only built for/installed for .NET Framework.